datacapturing/src/main/java/de/cyface/datacapturing/PongReceiver.java

```diff
@@ -85,8 +87,8 @@ public PongReceiver(@NonNull final Context context, @NonNull final String device
         this.deviceId = deviceId;
     }
 
-    // TODO: This should be called ping and receive, but maybe more meaningful names like: areYouRunning and iAmRunning
-    // would be more readable.
+    // TODO [CY-3950]: This should be called ping and receive, but maybe more meaningful names like: areYouRunning and
+    // iAmRunning would be more readable.
```

Comment:
But violates standard Java coding style.

Reply:
That's an old comment of yours. but I updated it